### PR TITLE
multiarch/add jbtrystram ssh jkey

### PR DESCRIPTION
**Pull request by jbtrystram**
fixup previous commit : this should be an rsa key !

### CODE REVIEW

**Comment by travier** [comment]
Needs a rebase :)

File: multi-arch-builders/builder-common.bu [hunk 1]
```diff
@@ -27,7 +27,8 @@ passwd:
...
-        - ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIN4IzlQKAbfMEUsWrPG4JDJVCJDRHOC+U8ZbYRXC4K0p jbtrystram@redhat.com
+        - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQCeVRo3murzVdV9OqZaDdn92Hor4hOejLlyY6AlydBv+CeXidz5Vdil7dBO6gX+QhbXusdKTDE7sb554C73ezssXyoC6ttfZE3j+bsVvy5BjlAgJ5jWEfBjR2/AJ1kjLhwg6fSroZpQHNmpPV3JOMKefRFG/yML6fFx6szGlxkOOesGCGMZTEEgM2m+TwyIEwb0bNdc/Qakir1kb/qOU2InfuDAX+gpjQVNYLSUbdiXQyqrCGFmeM0PhwgWmBJBrCLSO0a/QMtqnI/z9XRbjEGkkwQmVdDBDcxnO5mlqro5bTvAPKtbuM/pUjdOvMny1OQIgNFs5YJIMlKjQUoO/ZyVlndtldSDrE2L4hLOQu8i6Qx22FZKCco7h1yvCkJ0xlu4hKe4EqiueMScLjfYdA3IUH1e5Pi+F6v55Bcq6jxRUDTpbhFx94d6JznC8S40R/byvRAbsfvBS61XOtxEoA9orPooGBSW9AcP5ZTbSiAktQHUXih7XaUWIv7PnFMc60s= jbtrystram@redhat.com 
+>>>>>>> b568d8a (multiarch/add jbtrystram ssh jkey)
```
Comment (ravanelli) [inline multi-arch-builders/builder-common.bu:31]: Forgot to remove:
- >>>>>>> b568d8a (multiarch/add jbtrystram ssh jkey)

Comment (jbtrystram) [reply multi-arch-builders/builder-common.bu:31]: wooops

File: multi-arch-builders/builder-common.bu [hunk 2]
```diff
@@ -27,7 +27,7 @@ passwd:
...
-        - ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIN4IzlQKAbfMEUsWrPG4JDJVCJDRHOC+U8ZbYRXC4K0p jbtrystram@redhat.com
+        - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQCeVRo3murzVdV9OqZaDdn92Hor4hOejLlyY6AlydBv+CeXidz5Vdil7dBO6gX+QhbXusdKTDE7sb554C73ezssXyoC6ttfZE3j+bsVvy5BjlAgJ5jWEfBjR2/AJ1kjLhwg6fSroZpQHNmpPV3JOMKefRFG/yML6fFx6szGlxkOOesGCGMZTEEgM2m+TwyIEwb0bNdc/Qakir1kb/qOU2InfuDAX+gpjQVNYLSUbdiXQyqrCGFmeM0PhwgWmBJBrCLSO0a/QMtqnI/z9XRbjEGkkwQmVdDBDcxnO5mlqro5bTvAPKtbuM/pUjdOvMny1OQIgNFs5YJIMlKjQUoO/ZyVlndtldSDrE2L4hLOQu8i6Qx22FZKCco7h1yvCkJ0xlu4hKe4EqiueMScLjfYdA3IUH1e5Pi+F6v55Bcq6jxRUDTpbhFx94d6JznC8S40R/byvRAbsfvBS61XOtxEoA9orPooGBSW9AcP5ZTbSiAktQHUXih7XaUWIv7PnFMc60s= jbtrystram@redhat.com
```
Comment (marmijo) [inline multi-arch-builders/builder-common.bu:30]:
```suggestion
        - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQCeVRo3murzVdV9OqZaDdn92Hor4hOejLlyY6AlydBv+CeXidz5Vdil7dBO6gX+QhbXusdKTDE7sb554C73ezssXyoC6ttfZE3j+bsVvy5BjlAgJ5jWEfBjR2/AJ1kjLhwg6fSroZpQHNmpPV3JOMKefRFG/yML6fFx6szGlxkOOesGCGMZTEEgM2m+TwyIEwb0bNdc/Qakir1kb/qOU2InfuDAX+gpjQVNYLSUbdiXQyqrCGFmeM0PhwgWmBJBrCLSO0a/QMtqnI/z9XRbjEGkkwQmVdDBDcxnO5mlqro5bTvAPKtbuM/pUjdOvMny1OQIgNFs5YJIMlKjQUoO/ZyVlndtldSDrE2L4hLOQu8i6Qx22FZKCco7h1yvCkJ0xlu4hKe4EqiueMScLjfYdA3IUH1e5Pi+F6v55Bcq6jxRUDTpbhFx94d6JznC8S40R/byvRAbsfvBS61XOtxEoA9orPooGBSW9AcP5ZTbSiAktQHUXih7XaUWIv7PnFMc60s= jbtrystram@redhat.com
```

**Review by travier — APPROVED** [review]

File: multi-arch-builders/builder-common.bu (hunk 2 above)
Comment (marmijo) [reply multi-arch-builders/builder-common.bu:30]: Some trailing whitespace here

**Review by marmijo — COMMENTED** [review]
One comment, otherwise LGTM

**Review by travier — DISMISSED** [review]
As Michael said :)

**Comment by jbtrystram** [comment]
Updated

**Review by marmijo — APPROVED** [review]

**Pull request merged**